src/java.base/share/classes/java/security/KeyStore.java

```diff
@@ -1029,12 +1029,14 @@ public final String getType()
      * @param alias the alias name
      * @return an unmodifiable {@code Set} of attributes, possibly empty
```

Comment:
If the alias does not exist, then is it definitely always empty? I think the word "possibly" here may be a bit misleading. Maybe reword this as:
"an unmodifiable {@code Set} of attributes." This set is empty if the given alias does not exist or there are no attributes associated with the alias. This set may also be empty for PrivateKeyEntry or SecretKeyEntry entries that contain protected attributes and are only available through the ... "

Reply:
Updated as suggested.